website_snippet_anchor/templates/snippets.xml

```diff
@@ -4,12 +4,11 @@
 
 <odoo>
 
-<template id="snippet_options" inherit_id="web_editor.snippet_options">
-    <xpath expr=".">
+<template id="snippet_options" inherit_id="website.snippets">
+    <xpath expr="//div[@id='snippet_options']" position="inside">
         <div data-js="anchor"
              data-selector="section, .carousel">
             <li data-anchor_ask=""
-                data-only="click"
```

Comment:
You should leave this line if you want to avoid `hover` events. However, there are still the `reset` ones... I think this is a bug in Odoo, not sure if still happening in v11 TBH.